src/components/pages/PendingApplications/PendingApplication.js

```diff
@@ -42,6 +89,11 @@ const columns = [
   },
 ];
 
+// Displays shape of table data
+const onChange = (pagination, filters, sorter, extra) => {
+  console.log('params', pagination, filters, sorter, extra);
+};
+
 const PendingApplications = () => {
   const [applications, setApplications] = useState([]);
   const [displayModal, setDisplayModal] = useState(false);
```

Comment:
I've learned that the best naming conventions for booleans uses 'is' to be the most readable (ex. isVisible). 'displayModal' implies it's a function that is opening up the modal